sdks/java/core/src/test/java/org/apache/beam/sdk/io/WriteTest.java

```diff
@@ -289,6 +292,19 @@ public void populateDisplayData(DisplayData.Builder builder) {
     assertThat(displayData, hasDisplayItem("numShards", 1));
   }
 
+  @Test
+  public void testWriteUnbounded() {
```

Comment:
Expected: (an instance of java.lang.IllegalArgumentException and exception with message a string containing "Write can only be applied to a Bounded PCollection")
     but: an instance of java.lang.IllegalArgumentException <java.lang.IllegalStateException: Producer unknown for input: <unnamed> [PCollection]> is a java.lang.IllegalStateException
Stacktrace was: java.lang.IllegalStateException: Producer unknown for input: <unnamed> [PCollection]
 at com.google.common.base.Preconditions.checkState(Preconditions.java:199)
 at org.apache.beam.sdk.runners.TransformHierarchy.addInput(TransformHierarchy.java:79)
 at org.apache.beam.sdk.Pipeline.applyInternal(Pipeline.java:395)